opal/mca/pmix/pmix3x/pmix/src/tools/pmix_info/support.c

```diff
@@ -1327,13 +1327,11 @@ void pmix_info_do_config(bool want_all)
     char *debug;
     char *have_dl;
     char *symbol_visibility;
-    char *manpages;
 
     /* setup the strings that don't require allocations*/
     debug = PMIX_ENABLE_DEBUG ? "yes" : "no";
     have_dl = PMIX_HAVE_PDL_SUPPORT ? "yes" : "no";
     symbol_visibility = PMIX_HAVE_VISIBILITY ? "yes" : "no";
-    manpages = PMIX_ENABLE_MAN_PAGES ? "yes" : "no";
```

Comment:
Do you want to just hard-code this output to `no`?

Reply:
I'd rather not as it implies that one could enable those pages - the fact is that the pages do not exist.